modules/stanford_events_importer/tests/src/Unit/StanfordEventsImporterTest.php

```diff
@@ -105,5 +105,34 @@ public function testParseXML() {
     $this->assertEquals("Class", $result[19]);
   }
 
+  /**
+   * Make sure exception passes back false.
+   * @return [type] [description]
+   */
+  public function testFetchXMLException() {
+    $client = new BadClient();
```

Comment:
you can do this with a mock object.
```
$client = $this->createMock(Client::class);
$client->method('get')->willThrow(new Exception('hey there'));
```

Reply:
I was having issues with that and prophecy last night. For whatever reason, the type requirements were not working and it was recognizing the passed in object as a mock object and not as the excepted GuzzleInterface. I should come back to that.